.github/workflows/post-to-slack/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Post To Slack Created RC Release
+description: Post a message to a Slack channel
+
+inputs:
+  SLACK_BOT_TOKEN:
+    description: 'The Slack token to use'
+    required: true
+  SLACK_TAG_CHANNELS:
+    description: 'The Slack channel to post to'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+      - name: Post to a Slack channel
+        uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: ${{ inputs.SLACK_TAG_CHANNELS }}
+          payload: |
```

Comment:
The payload is very specific to releases and even RC releases - can we generalize it?

Reply:
As we discussed, I moved the `PAYLOAD` as input for the composite from `create-rc-tag`